### ISSUE TITLE
picture environment

### ISSUE THREAD

**Issue by jdmichaud** [Khan/KaTeX#1346]
Any chance of katex supporting the `picture` environment in the foreseeable future.

**Comment by kevinbarabash** [comment]
That's kind of cool.  I don't think we'll support this in the short term.  I'd like us to have a feasible way developing this an extension as opposed to doing in directly in KaTeX proper.  The rationale being that this feature will probably increase KaTeX's size considerably, but probably won't be used by very many people.

**Comment by edemaine** [comment]
Agreed, this would be a cool extension! But it would likely big, so would make sense as one. (And it would probably need a champion to make it happen, as it's probably quite a bit of work.)

**Comment by ylemkimon** [comment]
Note `\includegraphics` has been added in #1620.

**Comment by kevinbarabash** [comment]
This should probably be part of contrib.  I think this could be split up into multiple PRs:
- add support for parsing things like `(2,3)`
- add basic support for `\begin{picture}`
- add support for `\put` and one basic command, maybe `\line`
- add the rest of the commands, `\circle`, `\oval`, etc.

**Comment by edemaine** [comment]
In principle, #2085's delimiter parsing should make parsing things like `(2,3)` relatively easy (though I haven't looked at it closely enough to be sure). `\def\put(#1,#2)#3...`
